### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by braydenc303**
Finally got the email validation working properly and passing tests.

### CODE REVIEW

File: src/app/new-user/new-user.page.ts [hunk 1]
```diff
@@ -50,6 +50,7 @@ export class NewUserPage implements OnInit {
 	};
 
   	ngOnInit() {
+		  console.log(this);
```
Comment (haxwell) [inline src/app/new-user/new-user.page.ts]: remove the console.log.. we limit superfluous console.logs in prod

File: src/app/new-user/new-user.page.ts [hunk 2]
```diff
@@ -70,32 +71,41 @@ export class NewUserPage implements OnInit {
...
+		} else if ((this.validations_form.controls.email.status === "VALID" && /^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9]+([\.]{1})([a-zA-Z0-9]{2,3})$/.test(this.validations_form.controls.email.value))) {
+			console.log('valid email')
 			this.validation_messages.phone[1] = { type: 'validCountryPhone', message: 'You have entered a valid email address. Please clear this field OR provide a 10 digit phone number.' };
 			this.validation_messages.email[1] = { type: 'pattern', message: 'Please enter a valid email, OR a ten digit phone number.' };
+		} else if ((this.validations_form.controls.email.status === "VALID" && !(/^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9]+([\.]{1})([a-zA-Z0-9]{2,3})$/.test(this.validations_form.controls.email.value)))) {
+			console.log('invalid email with valid status');
+			this.validation_messages.phone[1] = { type: 'validCountryPhone', message: 'Please enter a ten digit phone number, OR a valid email.' };
+			this.validation_messages.email[1] = { type: 'pattern', message: 'Please enter a valid email, OR a ten digit phone number.' };
 		} else {
+			console.log('case 5');
```
Comment (haxwell) [inline src/app/new-user/new-user.page.ts]: remove console logs.. use unit tests, instead, to be sure it is working.

**Comment by haxwell** [comment]
just wanted to poke you on this.. once we get those console.logs removed I can merge this.

**Comment by braydenc303** [comment]
Sorry about that.

**Pull request merged**